yaksh/views.py

```diff
@@ -1796,6 +1796,69 @@ def download_assignment_file(request, quiz_id, question_id=None, user_id=None):
 
 @login_required
 @email_verified
+def upload_users(request, course_id):
```

Comment:
Because its a django request.
And course details have send mail, clone course, enroll, reject etc. all are separate django views,
none are in course details view!
And no harm in having a separate view.